### PR TITLE
Wildfire - updated documentation for detonating script files

### PR DESCRIPTION
**Pull request by pzhou-splunk**
PAPP-22000 PAPP-22001 - updated Wildfire documentation to note that when detonating script files, the filename parameter is required or else a 418 error will be returned

Closes PAPP-22000

### CODE REVIEW

File: readme.html [hunk 1]
```diff
@@ -31,6 +31,13 @@
 	</ul>
 </p>
 
+<p><b>Detonate File: Filename Parameter</b></p>
+  <ul>
+    <li>According to the Wildfire documentation: "When submitting supported script files, you must specify an accurate filename using the context parameter, otherwise WildFire is unable to parse the file and returns a 418 Unsupported File Type response."</li>
+    <li>Please see the following link for more information: <a href="https://docs.paloaltonetworks.com/wildfire/u-v/wildfire-api/submit-files-and-links-through-the-wildfire-api/submit-a-remote-file-to-wildfire-api.html" target="_blank">Wildfire API Documentation</a></li>
+  </ul>
+</p>
+
```
Comment (paulnawat) [inline readme.html:40]: Do we also need to update `readme.md` as well?

Comment (pzhou-splunk) [reply readme.html:40]: you can just update `readme.html`

[start-release](https://github.com/splunk-soar-connectors/wildfire/actions/workflows/start-release.yml) will update `readme.md` from the updated `readme.html`

Comment (joshquerinphantom) [reply readme.html:48]: ah thats what I initially thought. welp i updated it anyway :O

Comment (paulnawat) [reply readme.html:48]: ah I see. Thanks. Resolving this thread.

**Review by paulnawat — APPROVED** [review]

**Pull request merged**